osu.Game/Users/UserRankPanel.cs

```diff
@@ -0,0 +1,212 @@
+                                                                CreateFlag(),
+                                                            }
+                                                        }
+                                                    },
+                                                    new Drawable[]
+                                                    {
+                                                        CreateUsername().With(username =>
+                                                        {
+                                                            username.Anchor = Anchor.CentreLeft;
+                                                            username.Origin = Anchor.CentreLeft;
+                                                        })
+                                                    }
+                                                }
+                                            }
+                                        },
+                                        Empty()
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    new Container
+                    {
+                        Name = "Bottom content",
+                        Margin = new MarginPadding { Top = main_content_height },
+                        RelativeSizeAxes = Axes.X,
+                        AutoSizeAxes = Axes.Y,
+                        Padding = new MarginPadding { Left = 80, Vertical = padding },
+                        Child = new GridContainer
+                        {
+                            RelativeSizeAxes = Axes.X,
+                            AutoSizeAxes = Axes.Y,
+                            ColumnDimensions = new[]
+                            {
+                                new Dimension(),
+                                new Dimension()
+                            },
+                            RowDimensions = new[] { new Dimension(GridSizeMode.AutoSize) },
+                            Content = new[]
+                            {
+                                new Drawable[]
+                                {
+                                    new ProfileValueDisplay(true)
+                                    {
+                                        Title = UsersStrings.ShowRankGlobalSimple,
+                                        Content = User.Statistics?.GlobalRank?.ToLocalisableString("\\##,##0") ?? (LocalisableString)"-"
+                                    },
+                                    new ProfileValueDisplay(true)
+                                    {
+                                        Title = UsersStrings.ShowRankCountrySimple,
+                                        Content = User.Statistics?.CountryRank?.ToLocalisableString("\\##,##0") ?? (LocalisableString)"-"
+                                    }
```

Comment:
I dunno. These are never going to update. I'd say they need to update as a minimum requirement to have this added, else it's misleading to the user.

Reply:
Sure, I'll look into it

Reply:
There probably needs to be a global event which is triggered by the results screen when a change to statistics happens, which can schedule a future update of this card (next time the panel is opened).

Probably out of scope for this PR, but I'd hope to have that as a prereq or follow-up PR so they can be merged together.

Reply:
Could updating `APIAccess.LocalUser` bindable (and binding the card updates to it) be an option here? To me it feels like updating all of the user info might be better than just statistics

Reply:
You're ~probably~ definitely going to run into the fact that `APIUser` equality will cause a no-op when you update that bindable.

Maybe separating `Statistics` into a separate bindable on `APIAcess` if that is enough to make things work? Then having results screen trigger a `apiAccess.RefreshLocalUser()` or something, which updates the bindable statistics.